src/widgets/constants.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
😱

Reply:
Following @rubenmoya constants.. 🆗 , but there are a bunch of them we could add here 😬  (it is not necessary to add them now!)

Reply:
About the constants, I think we should have only one `constants.js` file, in the root folder (`src`), and use it only for constants which will be shared across the project (map types, layer types, etc). If the constant is only used in one file, I suggest to keep it on that file. If the `constants.js` files starts getting big, we could always split it in multiple files in a `/constants` folder.